R/oolong_tm.R

```diff
@@ -154,7 +161,18 @@
         V <- length(input_model@terms)
         terms <- t(topicmodels::terms(input_model, k = V))
         all_terms <- unique(as.vector(terms[,seq_len(n_top_terms)]))
+    } else if ("BTM" %in% class(input_model)){
+        K <- input_model$K
+        V<-input_model$W
+        terms<-t(apply(input_model$phi, MARGIN=2, FUN=function(x){
+            x <- data.frame(token = names(x), probability = x)
+            x <- x[order(x$probability, decreasing = TRUE), ]
+            x<-x$token
+            head(x,n=length(x))
+        })) 
+        all_terms<-rownames(input_model$phi)
```

Comment:
This line is not 100% correct because it doesn't respect the `n_top_terms` argument.

Similar to the processing of other topic model objects, a like this would do:

```r
all_terms <- unique(as.vector(terms[,seq_len(n_top_terms)]))
```

Reply:
Got it fixed, it is acutally exactly analogous since the terms object is the same. I pushed it to the fork. Tell me if you are through with the review, I will do the respecitve changes and PR.